.github/CONTRIBUTING.md

```diff
@@ -18,32 +18,54 @@ If neither of these seem appealing, please post on our [gitter channel](https://
 
 Before you submit code, please talk to us via the issue tracker so we know you are working on it and also let us know your approach, to be sure that you are thinking on right direction.
 
-Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. To submit code, follow these steps:
+Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. 
 
-1. Create a new branch off of development. Select a descriptive branch name. The commands below will help you to get the latest version of the code from the upstream(master branch).
+To submit code, follow these steps:
 
-        git fetch upstream
-        git checkout master
-        git merge upstream/master
-        git checkout -b your-branch-name
+1. Create a new branch off of master. Select a descriptive branch name. If you have not added the upstream then run this command:
+
+```
+git remote add upstream https://github.com/Cloud-CV/EvalAI-ngx
+```
+and to verify :
+
+```
+git remote -v
+```
+
+The commands below will help you to get the latest version of the code from the upstream(master branch). 
+
+```
+git fetch upstream
+git checkout master
+git merge upstream/master
+git checkout -b your-branch-name
+```
 
 2. Commit and push code to your branch:
 
     - Commits should be self-contained and contain a descriptive commit message.
-    - Please make sure your code is well-formatted and adheres to PEP8 conventions (for Python) and the airbnb style guide (for JavaScript). For others (Lua, prototxt etc.) please ensure that the code is well-formatted and the style consistent.
+    
+    - Please make sure your code is well-formatted and adheres to PEP8 conventions (for Python) and the airbnb style guide (for               JavaScript). For others (Lua, prototxt etc.) please ensure that the code is well-formatted and the style consistent.
+    
     - Please ensure that your code is well tested.
+    
     - If you have to check for any linting issues, run the following command before creating the pull request:
         ```ng lint```
+    
```

Comment:
We can also write about running the testcases locally using `ng test` command

Reply:
Done.